flake.nix

```
@@ -6,176 +6,299 @@
     systems.url = "github:nix-systems/default";
   };
 
-  outputs = {
-    self,
-    nixpkgs,
-    systems,
-  }: let
-    go125Version = "1.24.7";
-    goHash = "sha256-Ko9Q2w+IgDYHxQ1+qINNy3vUg8a0KKkeNg/fhiS0ZGQ=";
-    eachSystem = f:
-      nixpkgs.lib.genAttrs (import systems) (system:
-        f (import nixpkgs {
-          system = system;
-          overlays = [
-            (final: prev: {
-              go_1_24 = prev.go_1_24.overrideAttrs {
-                version = go125Version;
-                src = prev.fetchurl {
-                  url = "https://go.dev/dl/go${go125Version}.src.tar.gz";
-                  hash = goHash;
-                };
-              };
-            })
-          ];
-        }));
-  in {
-    formatter = eachSystem (pkgs: pkgs.nixpkgs-fmt);
-
-    packages = eachSystem (pkgs: {
-      default = pkgs.buildGo124Module {
-        pname = "tsidp";
-        version =
-          if (self ? shortRev)
-          then self.shortRev
-          else "dev";
-        src = pkgs.nix-gitignore.gitignoreSource [] ./.;
-        ldflags = let
-          tsVersion = with builtins;
-            head (match ".*tailscale.com v([0-9]+\.[0-9]+\.[0-9]+-?[a-zA-Z]?).*" (readFile ./go.mod));
-        in [
-          "-w"
-          "-s"
-          "-X tailscale.com/version.longStamp=${tsVersion}"
-          "-X tailscale.com/version.shortStamp=${tsVersion}"
-        ];
-        vendorHash = "sha256-obtcJTg7V4ij3fGVmZMD7QQwKJX6K5PPslpM1XKCk9Q="; # SHA based on vendoring go.mod
-      };
-    });
-
-    overlays.default = final: prev: {
-      tsidp = self.packages.${prev.stdenv.hostPlatform.system}.default;
-    };
+  outputs =
+    {
+      self,
+      nixpkgs,
+      systems,
+    }:
+    let
+      go125Version = "1.24.7";
```

```suggestion
      goVersion = "1.24.7";
```

This is my fault, but can you clear this up while this is being edited?